<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SinexDialog</class>
 <widget class="QDialog" name="SinexDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sinex Metadata</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressource.qrc">
    <normaloff>:/gui/html/logo_comp.png</normaloff>:/gui/html/logo_comp.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>File Agency Code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="dataAgencyEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="fileAgencyEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Data Agency Code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Obs Time (yyyy-mm-dd hh:mm:ss) </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateTimeEdit" name="obsTimeEdit">
     <property name="maximumDateTime">
      <datetime>
       <hour>21</hour>
       <minute>59</minute>
       <second>59</second>
       <year>9999</year>
       <month>12</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="maximumTime">
      <time>
       <hour>21</hour>
       <minute>59</minute>
       <second>59</second>
      </time>
     </property>
     <property name="displayFormat">
      <string notr="true">yyyy-MM-dd hh:mm:ss</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="timeSpec">
      <enum>Qt::UTC</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileAgencyEdit</tabstop>
  <tabstop>dataAgencyEdit</tabstop>
 </tabstops>
 <resources>
  <include location="../ressource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SinexDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SinexDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
